pom.xml

```diff
@@ -23,7 +23,7 @@
 		<amazon-kinesis-client.version>1.13.3</amazon-kinesis-client.version>
 		<amazon-kinesis-producer.version>0.14.1</amazon-kinesis-producer.version>
 		<dynamodb-stream.version>1.5.2</dynamodb-stream.version>
-		<localstack.version>0.1.22</localstack.version>
+		<localstack.version>0.2.2</localstack.version>
```

Comment:
Are you sure that we can simply bump such a version without changes to tests who use Local Stack?
And: did you really test the project against this Local Stack version?

Reply:
They were working on my machine with the changes you can see to `ExtendedDockerTestUtils` and `KinesisBinderTests` you can find below. I initially thought of using LocalStack to add some tests but ended up not needing it. That being said my machine is a Linux box, maybe it's worth testing in a Mac as well.